Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
-    railties (6.0.0)
-      actionpack (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails-html-sanitizer (1.3.0)
+      loofah (~> 2.3)
+    railties (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
-    rake (12.3.3)
-    thor (0.20.3)
+    rake (13.0.1)
+    thor (1.0.1)
```

Comment:
### [thor](http://whatisthor.com/)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.20.3 → 1.0.1

[_[change-log](https://github.com/erikhuda/thor/blob/master/CHANGELOG.md), [source-code](https://github.com/erikhuda/thor/tree/v1.0.1)_]





<details>
<summary>Commits</summary>

A change of **114** commits. See the full changes on [the compare page](https://github.com/erikhuda/thor/compare/v0.20.3...v1.0.1).

These are the first 10 commits:
- (b2b1e7a) [Add Thor.deprecation_warning](https://github.com/erikhuda/thor/commit/b2b1e7aedf721ff069292f909acccb01d9a1d858)
- (7d199c0) [Always check default type. Deprecate incompatible types with…](https://github.com/erikhuda/thor/commit/7d199c0367997939857a3c3d96209515239989e3)
- (c7e38fd) [Add Thor.deprecation_warning](https://github.com/erikhuda/thor/commit/c7e38fd6d2fb41fdc7039644ae91f97a0ef64878)
- (cc8e8fd) [Deprecate relying on default exit_on_failure? [#621]](https://github.com/erikhuda/thor/commit/cc8e8fd59fd5a7c2909e80bfa1ff90dee1193d4b)
- (562629b) [Fix typo [ci skip]](https://github.com/erikhuda/thor/commit/562629bd0c1152c7ce8f341176c346134613ea3c)
- (1ec3eff) [Merge pull request #638 from yujideveloper/fix/typo](https://github.com/erikhuda/thor/commit/1ec3effc78f2a849924c2fb7835adcfe60985020)
- (371f2a3) [Fix erikhuda/thor#587](https://github.com/erikhuda/thor/commit/371f2a3c549e0bc57ac1f53d0050c340d71fb5e9)
- (4344761) [Merge pull request #639 from hugocorbucci/master](https://github.com/erikhuda/thor/commit/4344761bb555897f31ae7575ab7a60fe908893c8)
- (8c57f1d) [Do not require did_you_mean](https://github.com/erikhuda/thor/commit/8c57f1dee1d6cb9d0071c793eae05278ece4d8c3)
- (d3762d0) [Merge pull request #640 from quixoten/patch-1](https://github.com/erikhuda/thor/commit/d3762d0b89a80da488285d640b052f5276fac6bb)


</details>